_posts/2022-07-25-Building-CNF-Using-CNDP-Rust.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Building Cloud Native Network Functions using Rust
+author: CNDP Team
+excerpt_separator: <!--more-->

```

Comment:
You shouldn't need the excerpt_separator, as jekyll automatically cuts it off after the first paragraph which is a good place to end the excerpt.

Reply:
ok. I saw this excerpt_separator, in an older commit of CNDP blog. Will remove it
 https://raw.githubusercontent.com/CloudNativeDataPlane/cloudnativedataplane.github.io/f606df7f17458129dbe549461fcc1c574a8d4e76/_posts/2022-05-11-Enabling-A-Cloud-Based-Framework-For-Networking.md